projects/fsastorefront-e2e-cypress/cypress/integration/agents/obo-creates-quotes.e2e-spec.ts

```diff
@@ -0,0 +1,119 @@
+import * as register from '../../helpers/register';
+import { sellerIndira, stephenCustomer } from '../../sample-data/users';
+import testFilters from '../../support/filters';
+import * as homepage from '../../helpers/homepage';
+import { selectCustomer } from '../../helpers/checkout/obo';
+import * as checkout from '../../helpers/checkout/checkout-steps';
+import * as event from '../../helpers/checkout/insurance/event-checkout';
+import * as banking from '../../helpers/checkout/banking/checkout-banking';
+import * as currentAccount from '../../helpers/checkout/banking/current-account';
+import * as quotes from '../../helpers/my-account/quotes-and-applications';
+import { selectPaymentMethodInvoice } from '../../helpers/checkout/insurance/payment';
+import * as userIdentification from '../../helpers/checkout/banking/user-identification';
+
+testFilters([''], () => {
+  context('Check sample data and create quotes', () => {
+    before(() => {
+      cy.visit('/login');
+    });
+
+    it('Seller creates event quote for customer', () => {
```

Comment:
Test labels should begin with `Should` so it is readable like:
`it('Should enable seller to create event quote for customer'`, or something like that. That's why the method itself is called `it` 🙂